core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/drs/DrsResolver.scala

```diff
@@ -20,24 +20,17 @@ object DrsResolver {
    */
   val compactIdRegex: Regex = "(?<scheme>dos|drs)://(?<compactIdPrefix>(dg|drs)\\.[0-9a-z-]+):(?<path>.*)".r
   val hostNameRegex: Regex = "(?<scheme>dos|drs)://(?<hostname>[^?/:]+\\.[^?/:]+)/(?<path>.*)".r
-  def getProvider(uri: String): Option[String] =
-    try
-      Option(new URI(uri).getHost) match {
```

Comment:
URIs that get to this point should already start with `dos` or `drs` so there's no need to check for a URI host